testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/arquillian/AuthServerTestEnricher.java

```diff
@@ -164,16 +169,24 @@ public void initializeSuiteContext(@Observes(precedence = 2) BeforeSuite event)
                 throw new IllegalStateException("Frontend container (load balancer) misconfiguration");
             }
 
+            boolean authServerJboss = Boolean.parseBoolean(System.getProperty("auth.server.jboss.crossdc"));
+            String containerPreffix;
+            if (authServerJboss) {
```

Comment:
Can the whole condition be replaced with `containerPrefix = "auth-server-" + System.getProperty("node.name") + "-"`?